### DISCUSSION TITLE
JuliaCon 2023

### DISCUSSION THREAD

**Discussion by cncastillo** [MagneticResonanceImaging/MRIReco.jl#155] [Q&A] [answered]
Hi everyone! Next week I am going to JuliaCon 2023 (MIT, USA) to present my MRI simulation package ([KomaMRI.jl](https://github.com/cncastillo/KomaMRI.jl/)) and I think it could be interesting to give some publicity to the growing MRI ecosystem (`MagneticResonanceImaging`) to motivate more people to switch from MATLAB 😃. 

Do you mind if I show a few screenshots from our meetings? (maybe we can bring them back!)

Also, I wanted to add a diagram like the one in this presentation from @tknopp (Dic 2022). 
![image](https://github.com/MagneticResonanceImaging/MRIReco.jl/assets/5957134/8b37a44b-73d1-447e-b1e8-556f47ed5540)
Is there any significant update to the hierarchy that I should add? 

I would gladly receive any other ideas you have, or examples in which using Julia made a huge difference in your workflow, code speed, etc.

**Comment by tknopp** [comment]
> Do you mind if I show a few screenshots from our meetings?

From my side, sure please show some pictures.

> maybe we can bring them back!

In principle yes. I currently don't have time to focus on MRIReco.jl (development goes in waves on my side) but if some of you want to go ahead with this, just initiate it.

> Is there any significant update to the hierarchy that I should add?

There is one additional arrow from `LinearOperators` to `RegularizedLeastSquares`. Furthermore there are missing arrows for `MRIFingerprintingRecon`(https://github.com/MagneticResonanceImaging/MRFingerprintingRecon.jl/blob/master/Project.toml). Other than that it should be up to date.

**Comment by cncastillo** [comment, marked as answer]
> In principle yes. I currently don't have time to focus on MRIReco.jl (development goes in waves on my side) but if some of you want to go ahead with this, just initiate it.

Perfect. I would be happy to organize one when I am less busy. I also tag @mrikasper as we talked briefly about the meetings at ISMRM. 

In the meantime, I attached a summary of the MRI ecosystem (thanks to @beorostica). Hopefully, it is useful, and I did not forget too many packages. 

![MRIEcosystem](https://github.com/MagneticResonanceImaging/MRIReco.jl/assets/5957134/e5fc8f46-c812-43b3-bdf1-116f7d1cf8df)

**Reply by tknopp** [reply]
nice!

**Comment by mrikasper** [comment]
Great, thank you, Carlos @cncastillo, for the nice graphics, and congratulations on the MRM highlights of KomaMRI!

Maybe, to add to your list of related packages, together with Zhe Wu (Tim) @nbwuzhe and Alexander Jaffray @alexjaffray, we are working on gradient impulse response function (GIRF) corrections of trajectories in https://github.com/BRAIN-TO/MRIGradients.jl and have detailed a model-based reconstruction pipeline for the use case of spiral diffusion MRI in https://github.com/BRAIN-TO/GIRFReco.jl that also utilizes MRIReco.jl and MRIFieldMaps.

We recently submitted this work to JOSS.


Regarding the meeting of the Julia MRI community, I am totally for it, maybe let's do it in early September, after everyone is back from vacation?

All the best,
Lars

**Comment by cncastillo** [comment]
Nice, I am up for a meeting in September. In the meantime, I leave an updated version of the diagram that includes GIRFReco (as a png as the SVG did not export correctly).

![JuliaMRIEcosystem](https://github.com/MagneticResonanceImaging/MRIReco.jl/assets/5957134/10dcaea5-feda-4c9b-a0dd-6ab69d1132ee)

**Reply by mrikasper** [reply]
Nice update, thank you so much, Carlos! Let us know how JuliaCon went.
